jest.config.js

```diff
@@ -1,30 +1,65 @@
 const { jestDirAlias } = require('./dirAlias');
 
 module.exports = {
-  reporters: [
-    'default',
-    [
-      'jest-junit',
-      {
-        suiteName: 'Jest Tests',
-        outputDirectory: 'reports/jest',
-        uniqueOutputName: 'true',
-        ancestorSeparator: ' › ',
+  projects: [
+    {
+      reporters: [
```

Comment:
We could maybe add this for consistency between the 'projects' as well:
```suggestion
      displayName: 'Unit Tests',
      reporters: [
```
Think this means we get output like this which could be useful if we are running both types of test together:
```jest
 RUNS   Unit Tests  src/server/index.test.jsx
```

Reply:
Yeh I did think about adding that in. Will go for that then 👍